oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1256,7 +1257,7 @@ bool pldm::responder::oem_ibm_platform::Handler::watchDogRunning()
         isWatchDogRunning = pldm::utils::DBusHandler().getDbusProperty<bool>(
             watchDogObjectPath, watchDogEnablePropName, watchDogInterface);
     }
-    catch (const std::exception& e)
+    catch (const std::exception& /*e*/)
```

Comment:
error message here?

Reply:
not required error trace because it's checking watchdog is running or not, if not running then will return false.